cs-utilities/src/main/java/io/cloudslang/content/utilities/util/Descriptions.java

```diff
@@ -79,4 +79,18 @@
         public static final String BOOL_VALUE_DESC = "Boolean value to check.";
     }
 
-}
\ No newline at end of file
+    public static class Counter{
+        public static final String FAILURE_DESC = "There was an error while executing the request.";
+        public static final String SUCCESS_DESC = "The request was successfully executed.";
+        public static final String COUNTER_DESC = "Counts from one number to another number.";
+        public static final String RESULT_STRING_DESC = "The primary result is resultString, Result can also be used. result (All lower case) should not be used as it is the response code.";
```

Comment:
Shouldn't the primary result be returnResult?

Reply:
incorporated the review comment